downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -32,120 +32,111 @@ However, if you are using a version below 2201.0.0 (Swan Lake) and if you alread
 
 ## Install Ballerina
 
-If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
-
-## Migrate from Swan Lake Beta releases
->**Info:** If you have been using Swan Lake Beta releases, delete the `Dependencies.toml` files in your Ballerina packages when migrating to Ballerina 2201.4.0 (Swan Lake). 
-
-A few backward-incompatible changes have been introduced during the Swan Lake Beta program, and thereby, some of your existing packages may not compile with Ballerina 2201.4.0 (Swan Lake). Therefore, you need to delete the `Dependencies.toml` file to force the dependency resolver to use the latest versions of your dependencies. 
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
 
 ## Language updates
 
 ### New features
 
 #### Support for annotations on tuple members
 
-Annotations are now supported on tuple members. Tuple members can be annotated using annotations declared with the `field` attachment point.
+- Annotations are now supported on tuple members. Tuple members can be annotated using annotations declared with the `field` attachment point.
```

Comment:
Why are we making this point form? Not required.

```suggestion
Annotations are now supported on tuple members. Tuple members can be annotated using annotations declared with the `field` attachment point.
```

Reply:
Please check other places too.

Reply:
Basically this is content under a heading. We don't need separate bullet points. This is what we did previously also.